### DISCUSSION TITLE
晓晨 | /cn/2019/05/xiaochen/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1361] [Comments]
https://yihui.org/cn/2019/05/xiaochen/

[a comment by an automated account is not shown]

**Comment by LFG12** [comment]
一次偶然和袁凡姐交流，说报名和君商学院，然后袁凡姐说你也参加过和君，一个月后才想起，便来博客下搜索了“和君”二字，想来看一个其中的体会，前边都还有一笔带过，看来益辉哥是和君09年的，现在24年是和君十七期的毕业，在赣州的会昌王先生创建了一个和君小镇，近几年完的工，今年五一在那里举办了和君的毕业典礼，访问到这篇博客时，这个经历如此沉重，尤其在看到她的毕业报告时，对待生活积极、有干劲，想要亲自创出一片天地，她对和君同学的记忆收获写的如此细致真情，可没想到看到时却已别离。

在这儿的留言不知是否会再促起你不好的回忆，如果有说声抱歉。

事业与生活，在努力拼搏的同时，也要爱惜好自己。

愿那里充满美好。

**Reply by yihui** [reply]
福小小友你好，没想到你还溜达到我这儿来了。欢迎光临！

和君后来的这些发展我大概都清楚。我在二届时，说实话那里的氛围与我的性格并不是很相符，我感觉周围的人都过度热情，让我很有些社恐，所以有些集体活动我都没参加。王先生很想让我用统计的本领在他那儿折腾一番，但我没兴趣。我欣赏先生的理想，只是这个商学院的形式让我觉得有些喧嚣。不过换个角度看，这种形式对在校大学生来说也许有更好的激励作用。

健康是 1，其它外物如事业、名声、地位都是 0；有了 1，后面的 0 才有意义，否则再多 0 也依然是 0。可惜时光不能倒流，我们无法让十多年前的她那么拼。作为生者，也只有爱护好自己的身体了。

放心，留言并不会勾起什么不好的回忆。相反，你的留言让我再次回头翻翻年轻时自己和他人的文字，还会帮忙打扫掉一些中年的倦怠。是时候抄起键盘重新写起来了。

说来晓晨离开已整六年，我对时间的感受也日渐迟钝。

**Reply by LFG12** [reply]
益辉你好，今天是农历六月十四，我是小福，刚偶然发现昨天是你的生日，想与你发一个迟来的祝福。祝愿你和你的家人朋友都能常开心，共同体会健康又幸福的生活。
首先抱歉在这篇帖子下发祝福似乎有些格调差异，希望你不要介意。正巧就在这篇帖子下的缘起，邮箱里有了往来，应该是我第一次在益辉大大的博客里留言，前面可能是在袁凡、国政、统计之都的评论里有过互动。
当时联系益辉大大时我刚在和君小镇回来，对和君充满些好奇，能见识到很多学校以外的东西。
现在也顺利通过考验成为了和君18届香港班的一员。（PS和君对在读学生有优惠减免，可以免去原本1w5的学费），也算当上了谢大的师弟，因为后面三年在深圳读研，就报名的港深的班。研究生研究方向是点云和图像处理，计算机视觉，以后估计也要走计算机的路了。
今年大学毕业，暑期回高中看望老师时，刚得知一位老师的离世，愈发感慨于世事无常，对见面与相遇便也更加珍视。
更加发觉身体健康的重要性。
大城市都会有快节奏的生活，不知美国的生活节奏和国内相比是怎样，听说还是很有秩序的，不像国内大部分公司都不管劳动法。
祝愿各位都能健康的享受美好人生，祝好。

**Reply by yihui** [reply]
谢谢你！上次你留言时我在国内，登 Github 经常抽风，后来长假长途奔波后就忘了回复。我的生活节奏相对比较慢，毕竟是生活在美国的大农村。

我觉得理科生去和君看看挺好的，接受一些财经、管理、人文方面的熏陶。你读研的方向听起来不错啊，祝你这三年能学有所成，不过有没有所成倒也是次要的（学习是一辈子的事情，不必在意某一特定阶段是否有成果），更重要的是享受研学的时光。
